src/main/scala/za/co/absa/abris/examples/ConfluentKafkaAvroReader.scala

```diff
@@ -16,41 +16,41 @@
 
 package za.co.absa.abris.examples
 
-import java.util.Properties
-
+import org.apache.spark.sql.SparkSession
 import org.apache.spark.sql.functions.col
-import org.apache.spark.sql.{DataFrame, Dataset, Row}
-import za.co.absa.abris.examples.utils.ExamplesUtils._
+import za.co.absa.abris.config.AbrisConfig
 
 object ConfluentKafkaAvroReader {
 
-  private val PARAM_JOB_NAME = "job.name"
-  private val PARAM_JOB_MASTER = "job.master"
-  private val PARAM_PAYLOAD_AVRO_SCHEMA = "payload.avro.schema"
-  private val PARAM_LOG_LEVEL = "log.level"
-  private val PARAM_OPTION_SUBSCRIBE = "option.subscribe"
-
-  private val PARAM_EXAMPLE_SHOULD_USE_SCHEMA_REGISTRY = "example.should.use.schema.registry"
+  val kafkaTopicName = "test_topic"
 
   def main(args: Array[String]): Unit = {
 
-    // there is an example file at /src/test/resources/AvroReadingExample.properties
-    checkArgs(args)
+    val spark = SparkSession
+      .builder()
+      .appName("WriterJob")
+      .master("local[2]")
+      .getOrCreate()
 
-    val properties = loadProperties(args)
 
-    val spark = getSparkSession(properties, PARAM_JOB_NAME, PARAM_JOB_MASTER, PARAM_LOG_LEVEL)
+    spark.sparkContext.setLogLevel("INFO")
 
-    val stream = spark
+    val dataFrame = spark
       .readStream
       .format("kafka")
+      .option("kafka.bootstrap.servers", "localhost:9092")
+      .option("subscribe", kafkaTopicName)
       .option("startingOffsets", "earliest")
-      .addOptions(properties) // 1. this method will add the properties starting with "option."
-                              // 2. security options can be set in the properties file
+      .load()
 
-    val deserialized = configureExample(stream.load(), properties)
+    val abrisConfig = AbrisConfig
+      .fromConfluentAvro
+      .downloadReaderSchemaByLatestVersion
```

Comment:
Maybe sticking to `using` and `with` it would yield `.withReaderSchemaVersionLatest` or `.withReaderSchemaVersion(version)`?

Reply:
I would work, but it wouldnt create an english sentence anymore. Now you can almost read it as a sentence:
```
Download Reader Schema By Latest Version and TopicNameStrategy using Schema Registry.
```

Reply:
I think it would still read as English:

`from Confluent Avro with Reader Schema Version X with TopicNameStrategy Y using Schema Registry`

the idea was to just restrict the number of verbs.